### DISCUSSION TITLE
I can't compile it on Android Studio

### DISCUSSION THREAD

**Discussion by GochaGT** [NightscoutFoundation/xDrip#3085] [Bug reports]
I downloaded it, tried to run it, but it doesn't start, it gives an error and it doesn't compile in Android Studio.
I have updated Android Studio to the latest version, including Java.
Your sources are not working and please help me.

**Comment by Navid200** [comment]
What did you download?

**Reply by GochaGT** [reply]
https://github.com/NightscoutFoundation/xDrip

**Reply by Navid200** [reply]
If you want to compile, there is nothing to download from there.  You need to create a fork in GitHub.
Android Studio will load from your fork.  After you are done, you push from Android Studio, which will update your fork.
Then, you can create a pull request from your fork.

**Reply by GochaGT** [reply]
I do exactly the same method and it doesn't compile. If anyone has a solution please help me

**Reply by Navid200** [reply]
What fails?  Do you have a screenshot?

**Reply by GochaGT** [reply]
[error.txt](https://github.com/NightscoutFoundation/xDrip/files/12693059/error.txt)

**Reply by Navid200** [reply]
@GochaGT This post explains how to attach a screenshot: https://github.com/NightscoutFoundation/xDrip/discussions/1900
